Resources/Private/Language/de.locallang_db.xlf

```diff
@@ -23,6 +23,10 @@
 				<source>Image</source>
 				<target>Bild</target>
 			</trans-unit>
+			<trans-unit id="tx_tea_domain_model_product_tea.owner">
+				<source>FE user who created this record</source>
+				<target>FE-User, der diesen Datensatz erstellt hat</target>
```

Comment:
Maybe we should use the TYPO3 naming for the record type "Website-Benutzer" in German and "Website User" in English instead of "FE-User" and "FE user"? That would provide a streamlined UI/UX for editors.

Reply:
Thanks, done.